src/odemis/acq/stream/_sync.py

```diff
@@ -327,7 +337,7 @@ def acquire(self):
         executeAsyncTask(f, self._runAcquisition, args=(f,))
         return f
 
-    def _updateProgress(self, future, dur, current, tot, bonus=0):
+    def _updateProgress(self, future, dur, current, tot, bonus=0):  # TODO?
```

Comment:
For now `tot` is probably incorrect when integrationCount > 1. Maybe it's just a matter of updating the caller.